### ISSUE TITLE
vip: --no-timestamp default

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-flow#5505]
`cylc play` and `cylc vip` seem to have different defaults for the `--no-timestamp` option.

Also check `cylc vr`.

**Comment by wxtim** [comment]
I cannot replicate this. I've tried just using the commands, and also doing `LOG.critical(self.options.log_timestamp)` at the top of the main loop function.

**Comment by oliver-sanders** [comment]
```console
$ cylc vip one --no-detach
$ cylc validate /.../one
WARNING - Backward compatibility mode ON
Valid for cylc-8.2.0.dev
$ cylc install /.../one
WARNING - Backward compatibility mode ON
INSTALLED one/run1 from /.../one
$ cylc play one/run1

 ▪ ■  Cylc Workflow Engine 8.2.0.dev
 ██   Copyright (C) 2008-2023 NIWA
▝▘    & British Crown (Met Office) & Contributors

WARNING - Backward compatibility mode ON
INFO - Extracting job.sh to /.../cylc-
    run/one/run1/.service/etc/job.sh
INFO - Workflow: one/run1
WARNING - Backward compatibility mode ON
INFO - Scheduler: url=tcp://me:43030 pid=116514
INFO - Workflow publisher: url=tcp:/me:43019
INFO - Run: (re)start number=1, log rollover=1
INFO - Cylc version: 8.2.0.dev
INFO - Run mode: live
INFO - Initial point: 1
INFO - Final point: None
INFO - Cold start from 1
INFO - New flow: 1 (original flow from 1) 2023-05-05 11:01:50
INFO - [1/foo waiting(runahead) job:00 flows:1] => waiting
INFO - [1/foo waiting job:00 flows:1] => waiting(queued)
INFO - [1/foo waiting(queued) job:00 flows:1] => waiting
INFO - [1/foo waiting job:01 flows:1] => preparing
INFO - [1/foo preparing job:01 flows:1] submitted to localhost:background[116570]
INFO - [1/foo preparing job:01 flows:1] => submitted
INFO - [1/foo submitted job:01 flows:1] health: submission timeout=P1D, polling
    intervals=PT1H,...
INFO - [1/foo submitted job:01 flows:1] => running
INFO - [1/foo running job:01 flows:1] health: execution timeout=None, polling
    intervals=PT1H,...
INFO - [1/foo running job:01 flows:1] => succeeded
INFO - Workflow shutting down - AUTOMATIC
INFO - DONE
$ cylc play one --no-detach

 ▪ ■  Cylc Workflow Engine 8.2.0.dev
 ██   Copyright (C) 2008-2023 NIWA
▝▘    & British Crown (Met Office) & Contributors

2023-05-05T11:02:07+01:00 INFO - Extracting job.sh to /.../cylc-
    run/one/run1/.service/etc/job.sh
2023-05-05T11:02:07+01:00 INFO - Workflow: one/run1
2023-05-05T11:02:07+01:00 INFO - LOADING workflow parameters
2023-05-05T11:02:07+01:00 INFO - + workflow UUID =
    4e23cb89-d1ae-484f-92fd-404d0dff5941
2023-05-05T11:02:07+01:00 INFO - + UTC mode = False
2023-05-05T11:02:07+01:00 INFO - + cycle point time zone = +0100
2023-05-05T11:02:07+01:00 WARNING - Backward compatibility mode ON
2023-05-05T11:02:07+01:00 INFO - Scheduler: url=tcp://me:43004 pid=116919
2023-05-05T11:02:07+01:00 INFO - Workflow publisher: url=tcp://me:43030
2023-05-05T11:02:07+01:00 INFO - Run: (re)start number=2, log rollover=1
2023-05-05T11:02:07+01:00 INFO - Cylc version: 8.2.0.dev
2023-05-05T11:02:07+01:00 INFO - Run mode: live
2023-05-05T11:02:07+01:00 INFO - Initial point: 1
2023-05-05T11:02:07+01:00 INFO - Final point: None
2023-05-05T11:02:07+01:00 INFO - LOADING task run times
2023-05-05T11:02:07+01:00 INFO - + foo: 1
2023-05-05T11:02:07+01:00 INFO - LOADING task action timers
2023-05-05T11:02:07+01:00 INFO - + 1/foo poll_timer
2023-05-05T11:02:07+01:00 WARNING - 1/foo: task not found, skip
2023-05-05T11:02:07+01:00 INFO - + 1/foo ['try_timers', 'submission-retry']
2023-05-05T11:02:07+01:00 WARNING - 1/foo: task not found, skip
2023-05-05T11:02:07+01:00 INFO - + 1/foo ['try_timers', 'execution-retry']
2023-05-05T11:02:07+01:00 WARNING - 1/foo: task not found, skip
2023-05-05T11:02:07+01:00 INFO - Flows:
2023-05-05T11:02:07+01:00 WARNING - No active tasks matching: */*
2023-05-05T11:02:07+01:00 INFO - Workflow shutting down - AUTOMATIC
2023-05-05T11:02:07+01:00 INFO - DONE
```

**Comment by wxtim** [comment]
Closed by #5524
